src/modules/RA/DataLayer/components/DataLayerTabbedForm.js

```diff
@@ -229,4 +245,4 @@ const DataLayerTabbedForm = ({ viewList, ...props }) => (
   </>
 );
 
-export default withViewList(DataLayerTabbedForm);
+export default withStyles(styles)(translateRA(withViewList(DataLayerTabbedForm)));
```

Comment:
Import `compose()` function (from `src/utils/compose.js`) so you can do this:

```suggestion
export default compose(
  withStyles(styles),
  translateRA,
  withViewList,
)(DataLayerTabbedForm);
```

*Search for `export default compose` in project to take at look to previous use cases.*

Reply:
Done